### DISCUSSION TITLE
OCI compatibility for cQube

### DISCUSSION THREAD

**Discussion by manojsah13** [Sunbird-cQube/community#766] [General]
Hello Everyone,

Please suggest if cQube is compatible with Oracle cloud infrastructure.

**Comment by ananya-jain1** [comment]
Hi @manojsah13 - we are testing cQube with Oracle cloud infrastructure. It will be compatible with Oracle from April 17, 2023.

**Comment by RNarmatha** [comment]
Hi @manojsah13 - Yes, latest version of cQube is compatible with Oracle cloude Infrastructure
